jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Slider.java

```diff
@@ -109,10 +109,12 @@ public static int getInteger(String attr, WebElement el, int defaultValue) {
   }
 
   private String setNewStyle(String style, int value) {
-    style = style.replaceAll("[-?0-9]+", "");
-    int perInx = style.indexOf("%");
-    return style.substring(0, perInx) + 0 + style.substring(perInx, style.length()-2)
-        + value + style.substring(style.length()-2);
+    String styleLocal;
+    styleLocal= style.replaceAll("[-?0-9]+", "");
+    int perInx;
+    perInx = styleLocal.indexOf("%");
+    return styleLocal.substring(0, perInx) + 0 + style.substring(perInx, style.length() - 2)
+            + value + style.substring(style.length() - 2);
```

Comment:
It seems that style variable should be replaced by styleLocal on lines 116 and 117

Reply:
updated